src/style.css

```diff
@@ -1,3 +1,65 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
```

Comment:
please move it to reset file

Reply:
Not fixed, pls creat [reset.css file](https://blog.hubspot.com/website/css-reset) for such cases